src/common/hook/usePublishQueue/usePublishQueue.test.tsx

```diff
@@ -59,10 +68,35 @@ describe("usePublishQueue", () => {
     mockGetPublishingJobs.mockReturnValueOnce(sampleJobs);
     mockPublishJob.mockResolvedValue("tx-id");
     mockPublishJob.mockRejectedValueOnce(new Error("Some error"));
+    mockUploadToStorage.mockReturnValue(uploadSuccess);
     const { result } = renderHook(() => usePublishQueue(config, formEntires));
     await act(async () => {
       await result.current.publish();
     });
     expect(result.current.failedPublishedDocuments).toHaveLength(1);
   });
+
+  it("should publish correctly if uploaded document is working", async () => {
+    mockGetPublishingJobs.mockReturnValueOnce(sampleJobs);
+    mockPublishJob.mockResolvedValue("tx-id");
+    mockUploadToStorage.mockReturnValue(uploadSuccess);
+
+    const { result } = renderHook(() => usePublishQueue(config, formEntires));
+    await act(async () => {
+      await result.current.publish();
+    });
+
+    expect(result.current.publishedDocuments).toHaveLength(3);
+  });
+
```

Comment:
one common error that users will probably face is the exceed upload size limit error, not much point to write an additional test to handle it but can you try out the workflow to cause the error to see that it works as expected?

Reply:
was unable to figure out what goes wrong when we do a upload that exceeds the file limit, chrome says CORS error as well as (failed) without a http status - please make a story/issue to track the resolution of this

Reply:
ok